MaxText/layers/multi_token_prediction.py

```diff
@@ -136,4 +136,3 @@ def __call__(
     # Shape: [B, S, H]
     # --- Return Processed Hidden State ---
     return next_hidden_state
-  
```

Comment:
Why removing the new line at end of file?

Reply:
`bash code_style.sh` made it this way